pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistryServiceImpl.java

```diff
@@ -178,15 +178,8 @@ public CompletableFuture<List<CompletableFuture<SchemaAndMetadata>>> getAllSchem
     @NotNull
     public CompletableFuture<SchemaVersion> putSchemaIfAbsent(String schemaId, SchemaData schema,
```

Comment:
First I agree with you, that when the admin API creates Schema should check the schema Compatibility, but this method is putSchemaIfAbsent, if the schema exists, we should return the SchemaVersion, so I don't think we should change this method.

Reply:
@congbobo184 
I see. In the current implementation this is correct behavior, right? 

Maybe we should let `createSchema` method return the version of the schema, so users can see whether the schema is changed or not? If this change makes sense, I'll draft a proposal to introduce a new `createOrUpdateSchema` admin API, it can able to return the schema version.

Reply:
I think the existing schema should return the existing exception instead of checking compatibility. Create an existing schema, return a ConflictException, and the user will be confused. How do you feel about this?